packages/fxa-settings/src/pages/Signup/index.tsx

```diff
@@ -116,63 +114,25 @@ export const Signup = ({
         email,
         newPassword: '',
         confirmPassword: '',
-        age: '',
       },
     });
 
   const ftlMsgResolver = useFtlMsgResolver();
 
-  const localizedAgeIsRequiredError = ftlMsgResolver.getMsg(
-    'auth-error-1031',
-    'You must enter your age to sign up'
-  );
-  const localizedValidAgeError = ftlMsgResolver.getMsg(
-    'auth-error-1032',
-    'You must enter a valid age to sign up'
-  );
-
   const onFocus = () => {
     if (!isFocused) {
       logViewEvent('flow', `${viewName}.engage`, REACT_ENTRYPOINT);
       setIsFocused(true);
     }
   };
 
-  const onFocusAgeInput = () => {
-    setAgeCheckErrorText('');
-    if (!hasAgeInputFocused) {
-      GleanMetrics.registration.engage({ event: { reason: 'age' } });
-      setHasAgeInputFocused(true);
-    }
-  };
-
-  const onBlurAgeInput = () => {
-    getValues().age === '' && setAgeCheckErrorText(localizedAgeIsRequiredError);
-  };
-
   // TODO: Add metrics events to match parity with content-server in FXA-8302
   // The legacy amplitude events will eventually be replaced by Glean,
   // but until that is ready we must ensure the expected metrics continue to be emitted
   // to avoid breaking dashboards.
   const onSubmit = useCallback(
-    async ({ newPassword, age }: SignupFormData) => {
+    async ({ newPassword }: SignupFormData) => {
       GleanMetrics.registration.submit();
-      if (Number(age) < 13) {
-        // this is a session cookie. It will go away once:
-        // 1. the user closes the tab
-        // and
-        // 2. the user closes the browser
-        // Both of these have to happen or else the cookie
-        // hangs around like a bad smell.
-
-        // TODO: probably a better way to set this?
-        document.cookie = 'tooyoung=1;';
```

Comment:
Also remove the check for this cookie in the signup container?
+ should we also remove the cannot_create_account page since it will no longer be used?

Reply:
Oh missed that one, removed the check from signup container. I took a stab at removing the `cannot_create_account` page but ran into some issues, I'll file a follow up for that since it probably makes sesne to be its own thing.